example/comments.js

```diff
@@ -32,99 +32,101 @@ import FlatButton from 'material-ui/FlatButton';
 import { Toolbar, ToolbarGroup } from 'material-ui/Toolbar';
 import ChevronLeft from 'material-ui/svg-icons/navigation/chevron-left';
 import ChevronRight from 'material-ui/svg-icons/navigation/chevron-right';
-
+import LocalizedComponent from 'admin-on-rest/i18n/LocalizedComponent';
 export CommentIcon from 'material-ui/svg-icons/communication/chat-bubble';
 
-const CommentFilter = (props) => (
+const CommentFilter = LocalizedComponent(({ translate, ...props }) => (
     <Filter {...props}>
-        <ReferenceInput label="Post" source="post_id" reference="posts">
+        <ReferenceInput label={translate('post.name')} source="post_id" reference="posts">
             <SelectInput optionText="title" />
         </ReferenceInput>
     </Filter>
-);
+));
 
-const CommentPagination = ({ page, perPage, total, setPage }) => {
+const CommentPagination = LocalizedComponent(({ page, perPage, total, setPage, translate }) => {
     const nbPages = Math.ceil(total / perPage) || 1;
     return (
         nbPages > 1 &&
             <Toolbar>
                 <ToolbarGroup>
                 {page > 1 &&
-                    <FlatButton primary key="prev" label="Prev" icon={<ChevronLeft />} onClick={() => setPage(page - 1)} />
+                    <FlatButton primary key="prev" label={translate('aor.message.prev')} icon={<ChevronLeft />} onClick={() => setPage(page - 1)} />
                 }
                 {page !== nbPages &&
-                    <FlatButton primary key="next" label="Next" icon={<ChevronRight />} onClick={() => setPage(page + 1)} labelPosition="before" />
+                    <FlatButton primary key="next" label={translate('aor.message.next')} icon={<ChevronRight />} onClick={() => setPage(page + 1)} labelPosition="before" />
                 }
                 </ToolbarGroup>
             </Toolbar>
     );
-}
+});
 
 const cardStyle = {
     width: 300,
     minHeight: 300,
     margin: '0.5em',
     display: 'inline-block',
-    verticalAlign: 'top'
+    verticalAlign: 'top',
 };
-const CommentGrid = ({ ids, data, basePath }) => (
+
+const CommentGrid = LocalizedComponent(({ ids, data, basePath, translate }) => (
     <div style={{ margin: '1em' }}>
     {ids.map(id =>
         <Card key={id} style={cardStyle}>
             <CardHeader
-                title={<TextField record={data[id]} source="author.name" />}
-                subtitle={<DateField record={data[id]} source="created_at" />}
+                title={<TextField record={data[id]} source="author.name" label={translate('author.list.name')} />}
```

Comment:
My bad. As it's only one component (`<CardHeader`) and I saw two `label` occurrences, I thought it was used twice. But now I understand that it's in two different sub components, so no problem.